LayoutTest/Swift/DeviceConstants.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  DeviceConstants.swift
+//  LayoutTest
+//
+//  Created by Peter Livesey on 9/21/16.
+//  Copyright © 2016 LinkedIn. All rights reserved.
+//
+
+import Foundation
+
+/**
+ This file redefines all the constants in LYTDeviceConstants for use in Swift.
+ This wasn't needed in previous version of Swift because this conversion was done automatically.
+ */
+
+public let iPhone4Width = NSNumber(value: Double(LYTiPhone4Width))
```

Comment:
Do you need a NSNumber here because ultimately these will get put in a collection? I am kind of surprised that LYTViewSize doesn't just store CGFloats.

Seems like there should be some sort of namespace, either by defining these in an extension on NSNumber, or wrapping them in an enum.